### PR TITLE
✨ Add Comscore Translated Consent Parameter

### PR DESCRIPTION
**Pull request by wroberts-comscore**
Adding Comscore's specific consent value translated from consentState macro. 

- If consentState is sufficient, we want cs_ucfr=1

- If consentState is insufficient, we want cs_ucfr=0

- If consentState is otherwise unknown, we want cs_ucfr=   (literal blank value)

### CODE REVIEW

**Comment by wroberts-comscore** [comment]
@calebcordry anything I can do to further this along?

File: extensions/amp-analytics/0.1/vendors/comscore.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ const COMSCORE_CONFIG = jsonLiteral({
     'pageview':
       '${base}c1=2' +
       '&c2=${c2}' +
+      '&cs_ucfr=$IF($EQUALS(${consentState}, sufficient), 1)$IF($EQUALS(${consentState}, insufficient), 0)$IF($EQUALS(${consentState}, ), )' +
```
Comment (calebcordry) [inline extensions/amp-analytics/0.1/vendors/comscore.js]: I don't think the nested macros will currently work like this, have you tested?

Comment (wroberts-comscore) [reply extensions/amp-analytics/0.1/vendors/comscore.js]: Yeah, I tested it I can send you my test page if you'd like to see yourself

Comment (calebcordry) [reply extensions/amp-analytics/0.1/vendors/comscore.js]: Great! I believe you :) thanks for confirming.

**Comment by calebcordry** [comment]
Just restarted unrelated test flake, will keep an eye on it but feel free to ping if you see it pass.

**Comment by wroberts-comscore** [comment]
Going to attach my test page, just for posterity. Hopefully it helps someone down the line. Feel free to delete this if it's unwanted. 

[analytics-comscore-consent.amp.html.txt](https://github.com/ampproject/amphtml/files/3740214/analytics-comscore-consent.amp.html.txt)

**Comment by calebcordry** [comment]
@wroberts-comscore I tried restarting a few times, keeps flaking. Could you try rebasing to latest master? Thanks

**Comment by wroberts-comscore** [comment]
@calebcordry I just rebased, following this guide: https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

I'm not sure if it went through, how can I check if the rebase is now working on this PR?

**Pull request merged**